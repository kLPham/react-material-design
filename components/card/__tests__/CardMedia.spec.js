import React from 'react';
import { shallow } from 'enzyme';
import CardMedia from '../cardMedia';

describe('<CardMedia />', () => {
    it('should render', () => {
        const src = 'http://google.com/image.jpeg';
        const cardMedia = shallow(
            <CardMedia>
                <img src={src} />
            </CardMedia>,
    );
        expect(cardMedia).toMatchSnapshot();
    });
    it('should render with children', () => {
        const cardMedia = shallow(
            <CardMedia>Text</CardMedia>,
    );
        expect(cardMedia).toMatchSnapshot();
    });
});
