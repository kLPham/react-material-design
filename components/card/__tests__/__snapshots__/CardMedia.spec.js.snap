// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardMedia /> should render 1`] = `
<section
  className="mdc-card__media"
>
  <img
    src="http://google.com/image.jpeg"
  />
</section>
`;

exports[`<CardMedia /> should render with children 1`] = `
<section
  className="mdc-card__media"
>
  Text
</section>
`;
